kolibri/plugins/demo_server/assets/src/views/DemoServerIndex.vue

```diff
@@ -72,6 +68,19 @@
       KButton,
       KExternalLink,
     },
+    $trs: {
+      demoServerHeader: 'Welcome to the Kolibri demo site!',
+      demoServerP1: 'You are welcome to explore with any of the three primary user types:',
+      demoServerL1: "Learner (username: '{user}', password: '{pass}') or access as a guest",
+      demoServerL2: "Coach (username: '{user}', password: '{pass}')",
+      demoServerL3: "Admin (username: '{user}', password: '{pass}')",
+      demoServerP2:
+        'This online demo site includes the features and functionalities of the latest version, but only holds a sample of the content that has been made available for offline use in Kolibri. It is intended for demonstration purposes and any user data on here may be cleared periodically.',
+      demoServerP3:
+        'To learn more about using Kolibri, take a look at our and better understand the platform in an offline context:',
```

Comment:
Am I missing something here...? `...take a look at our and better...`

Reply:
thanks good catch!  slipped by @rtibbles ;-)

Reply:
Only human...

Reply:
you claim

Reply:
:smiling_imp: